src/backend/app.js

```diff
@@ -4,32 +4,139 @@ const router = express.Router();
 const path = require("path");
 
 const mealsRouter = require("./api/meals");
+const reservationsRouter = require("./api/reservations");
+
 const buildPath = path.join(__dirname, "../../dist");
 const port = process.env.PORT || 3000;
 const cors = require("cors");
 
-// For week4 no need to look into this!
-// Serve the built client html
+//to get connected to the database
+const knex = require("./database");
+
 app.use(express.static(buildPath));
 
 // Parse URL-encoded bodies (as sent by HTML forms)
 app.use(express.urlencoded({ extended: true }));
+
 // Parse JSON bodies (as sent by API clients)
 app.use(express.json());
 
 app.use(cors());
 
 router.use("/meals", mealsRouter);
+router.use("/reservations", reservationsRouter);
+
+//________________________________________________
+
+//-----week1-----
+
+const todayDate = new Date();
```

Comment:
I think it would be safer to place this inside the callbacks (the function you putt to `app.get()` ) - that way every time someone calls an endpoint they get the new date that is for the current moment. With this line I am afraid that it might run only once so the data you think is now will be in the past. In short - new date per request :)